settings.gradle

```diff
@@ -0,0 +1,6 @@
+include ':butterknife'
+include ':butterknife-annotations'
+include ':butterknife-compiler'
+include ':butterknife-sample'
+
+//rootProject.name = 'butterknife-parent'
```

Comment:
uncomment? otherwise I think module names will clash in AS.

also... newline :grimacing: